### PR TITLE
add io.github.pwr_solaar.solaar

### PR DESCRIPTION
**Pull request by proletarius101**
Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements](https://docs.flathub.org/docs/for-app-authors/requirements) and [App Maintenance](https://docs.flathub.org/docs/for-app-authors/maintanance) pages.
- [x] My pull request follows the instructions at [App Submission](https://docs.flathub.org/docs/for-app-authors/submission).
- [x] I am using only the minimal set of permissions. (If not, please explain each non-standard permission.)
- [x] All assets referenced in the manifest are redistributable by any party. If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. Link: https://github.com/pwr-Solaar/Solaar/issues/1540
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. (If not, explain why.)

Previous PR: https://github.com/flathub/flathub/pull/2984

### CODE REVIEW

**Comment by hfiguiere** [comment]
> I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. Link: https://github.com/najepaliya/kleaner

That project mentioned here is another submission in queue. Can you elaborate?

File: io.github.pwr_solaar.solaar.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+rename-icon: solaar
+rename-desktop-file: solaar.desktop
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.gtk.vfs.*
+  - --filesystem=xdg-run/gvfsd
+
```
Comment (bbhtt) [inline io.github.pwr_solaar.solaar.yaml:15]: Cleanup `/;ib/pkgconfig`

**Comment by barthalion** [comment]
bot, build io.github.pwr_solaar.solaar

**Comment by barthalion** [comment]
Hmm, yeah, it seems to be working fine on the latest Fedora and a unifying receiver. Let's see how it goes.

/merge

**Pull request closed without merging**